src/network/networkprotocol.h

```diff
@@ -20,6 +20,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #pragma once
 
 #include "util/string.h"
+#include "gettext.h"
```

Comment:
To avoid including "gettext.h" in "networkprotocol.h", you could move the strings to "clientpackethandler.cpp" where they are used.

Reply:
Done in https://github.com/minetest/minetest/pull/14842/commits/ce28d00a2194648ee7aa84aea3e1bba151b4a867